### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.1.1][IntelliJ][Wipro]: Unable to access "Context menu" with keyboard for Sign out dialog.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1519]
**Repro Steps:**

1. Create a new dynamic web project.
2. tab to and select Project
3. Try to access the "Context menu" using  "Shift + F10" with keyboard.

**Expected Result:**
User should be able to access the "Context menu" with keyboard.

**Actual Result:**
Unable to access "Context menu" with keyboard for sign out.

**Platform:**
IntelliJ IDEA 2017.3.4 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![signout context menu is not accessible](https://user-images.githubusercontent.com/22956130/36151202-3a82c102-107b-11e8-93b7-bd04a12d1873.png)

**Comment by Eskibear** [comment]
"Shift + F10" is not supposed to access context menu.

> Run/Debug :  Shift + F10 / F9

Ref:
https://resources.jetbrains.com/storage/products/intellij-idea/docs/IntelliJIDEA_ReferenceCard.pdf

**Comment by WiproPSAccTest** [comment]
Keyboard user are not able to access the context menu .
Can you please provide the shortcut key to open the context menu

**Comment by Eskibear** [comment]
Will be fixed in PR #1638 , "CONTEXT_MENU" key is supposed to be pressed. (usually the one next to your right-win key)

**Comment by Eskibear** [comment]
Already in latest dev build.

**Comment by WiproPSAccTest** [comment]
Verified the issue in V3.7.0. Issue fixed.
Please add closed tag
![1519-fixed](https://user-images.githubusercontent.com/22956130/39859416-6611d630-5457-11e8-8b7b-06121493882b.jpg)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749976
